networkx/generators/harary_graph.py

```diff
@@ -23,29 +23,28 @@
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def hnm_harary_graph(n, m, create_using=None):
-    """Returns the Harary graph with given numbers of nodes and edges.
+    r"""Return the Harary graph with given numbers of nodes and edges.
```

Comment:
Can you remind me why we need to change this doc_string to raw (with the `r` in front)?
Is it because we are using latex?

Also, why are we changing `Returns` to `Return`?  I thought we were moving toward using the gammar `Returns` on all the doc_strings. Seems like we switch back and forth over the years. Wish we just didn't bother. :)